ffi/README.md

```diff
@@ -60,6 +60,79 @@ cargo build -p firewood-ffi --features ethhash
 
 To support development in [Coreth](https://github.com/ava-labs/coreth), Firewood pushes static libraries for Ethereum-compatible hashing to [firewood-go-ethhash](https://github.com/ava-labs/firewood-go-ethhash) with `ethhash` enabled by default. To use Firewood's native hashing structure, you must still build the static library separately.
 
+## Configuration
+
+### Database Config
+
+A `Config` must be provided when creating the database. A default config is provided at `ffi.DefaultConfig()`:
+
+```go
+&Config{
+    NodeCacheEntries:     1000000,
+    FreeListCacheEntries: 40000,
+    Revisions:            100,
+    ReadCacheStrategy:    OnlyCacheWrites,
+}
+```
+
+A description of all available values, and the default if not set, is available below.
+
+#### Truncate - `bool`
+
+If set to `true`, an empty database will be created, overriding any existing file. Otherwise, if a file exists, that file will be loaded to cr:w
+eate the database. In either case, if the file doesn't exist, it will be created.
+
+*Default*: `false`
+
+#### Revisions - `uint`
+
+Indicates the number of committed roots accessible before the diff layer is compressed. Must be explicitly set if the config is specified to at least 2.
+
+#### ReadCacheStrategy - `uint`
+
+Should be one of `OnlyCacheWrites`, `CacheBrancheReads`, or `CacheAllReads`. In the latter two cases, writes are still cached.
```

Comment:
typo. Does golang doc have some way to avoid these?
```suggestion
Should be one of `OnlyCacheWrites`, `CacheBranchReads`, or `CacheAllReads`. In the latter two cases, writes are still cached.
```